backend/player.py

```diff
@@ -0,0 +1,24 @@
+from backend.position import Position
+from backend.piece import Piece
+class Player:
```

Comment:
you don't need the class player, player can be used as a simple string of "red" or "black"